tests/phpunit/CRM/Utils/HookTest.php

```diff
@@ -67,8 +67,9 @@ public function testRunHooks_merge() {
   }
 }
 
-/* --- Library of test hook implementations ---- */
```

Comment:
I suspect the phpcs advice in this file was misleading. For example, this comment described a section with nine different functions -- it's not documentation for the one function "hooktesta_civicrm_testRunHooks_outer". I think the way to appease phpcs here would be to make up documentation for the "hooktesta_civicrm_testRunHooks_outer" like "Implements hook_civicrm_testRunHooks_outer."

There's a similar issue down below -- it's significant that the library of test functions does *not* include a particular function. The non-existence of hooktestb_civicrm_testRunHooks_merge doesn't have any bearing on hooktestc_civicrm_testrunHooks_merge.